{{cookiecutter.project_name}}/steps/predict.py

```diff
@@ -11,7 +13,13 @@ def predict_batch(spark_session, model_uri, input_table_name, output_table_name)
     predict = mlflow.pyfunc.spark_udf(
         spark_session, model_uri, result_type="string", env_manager="conda"
     )
-    output_df = table.withColumn("prediction", predict(struct(*table.columns)))
+    output_df = (
+        table.withColumn("prediction", predict(struct(*table.columns)))
+        .withColumn("model_version", lit(model_version))
+        .withColumn("ts", lit(ts))
+        .withColumn("model_version", col("model_version").cast("int"))
+        .withColumn("ts", to_timestamp("ts"))
```

Comment:
Bit worried `ts` isn't descriptive enough. Does data monitoring want it to be called `ts`? If not I think we can expand it to call it like `batch_timestamp` 🙂

Reply:
The data monitoring example in the API reference is using `ts`, which is short for timestamp. 
It could be from batch inference or online serving I think.

Reply:
@vperiyasamy any chance we can make this `ts` column a bit more descriptive in the example as well 😛